CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 - Removed: async mode/option
 ([#107](https://github.com/postcss/postcss-import/pull/107))
+- Removed: bower is not currently supported, use custom resolver
+([#116](https://github.com/postcss/postcss-import/pull/116))
```

Comment:
fyi, you can just add bower support by it to the `path` option.
So I would use ``"bower_components" not supported by default anymore, use "path" option to add it back``.